src/irr/asset/CGLTFLoader.h

```diff
@@ -47,6 +47,17 @@ namespace irr
 					std::optional<std::string> copyright;
 				};
 
+				enum SGLTFPrimitiveTopology
+				{
+					SGLTFPT_POINTS,
+					SGLTFPT_LINES,
+					SGLTFPT_LINE_LOOP,
+					SGLTFPT_LINE_STRIP,
+					SGLTFPT_TRIANGLES,
+					SGLTFPT_TRIANGLE_STRIP,
+					SGLTFPT_TRIANGLE_FAN
```

Comment:
dont you need to assign them numbers?

Reply:
the docs say I do not need to

Reply:
so their values are basically 0,1,2,3,4,... ?

Reply:
?

Reply:
yeah